addons/test_mass_mailing/tests/test_mailing_statistics_sms.py

```diff
@@ -25,6 +25,31 @@ def setUpClass(cls):
             signature='--\nMarie'
         )
 
+        records = cls.records
+        partners = cls.partners
+        country_be_id = cls.env.ref('base.be').id
+        _country_us_id = cls.env.ref('base.us').id
+
+        for x in range(10):
+            partners += cls.env['res.partner'].with_context(**cls._test_context).create({
+                'name': 'Partner_1_%s' % (x),
+                'email': '_test_partner_1_%s@example.com' % (x),
+                'country_id': country_be_id,
+                'mobile': '045300%s%s99' % (x, x)
+            })
+            records += cls.env['mail.test.sms'].with_context(**cls._test_context).create({
+                'name': 'MassSMSTest_1_%s' % (x),
+                'customer_id': partners[x].id,
+                'phone_nbr': '045300%s%s44' % (x, x)
```

Comment:
all that could be moved in a small helper (a bit like '_create_mailing_test_records')